build.gradle

```diff
@@ -514,6 +517,8 @@ task evmToolStartScripts(type: CreateStartScripts) {
   defaultJvmOpts = [
     "-Dsecp256k1.randomize=false"
   ]
+  unixStartScriptGenerator.template = resources.text.fromFile("${projectDir}/besu/src/main/scripts/unixStartScript.txt")
+  windowsStartScriptGenerator.template = resources.text.fromFile("${projectDir}/besu/src/main/scripts/windowsStartScript.txt")
   doLast {
```

Comment:
${rootDir} instead maybe?

Reply:
The tasks are only in the root project.  The only place we use rootdir in the gradle scripts right now is in sub-projects.